app/assets/javascripts/features/confirm_dialog.js

```diff
@@ -0,0 +1,32 @@
+$(function(){
+  $(document).on('click', '.confirm-dialog-wrapper .close-dialog', function(ev){
+    $(this).closest('.confirm-dialog-wrapper').fadeOut(100);
```

Comment:
Ack no this is fine, just hadn’t read the rest of the code before I just read this bit